ui/page/privacy/manual_mixer_setup_page.go

```diff
@@ -122,65 +116,110 @@ func (pg *ManualMixerSetupPage) OnNavigatedTo() {
+										}),
+										layout.Rigid(func(gtx C) D {
+											card := pg.Theme.Card()
+											card.Color = pg.Theme.Color.Gray4
+											return card.Layout(gtx, func(gtx C) D {
+												gtx.Constraints.Min.X = gtx.Constraints.Max.X
+												gtx.Constraints.Min.Y = 110
+												return layout.Inset{Top: values.MarginPadding10, Left: values.MarginPadding16,
+													Right: values.MarginPadding16}.Layout(gtx, func(gtx C) D {
+													return layout.Inset{Top: values.MarginPaddingMinus15, Left: values.MarginPaddingMinus15,
+														Right: values.MarginPaddingMinus15, Bottom: values.MarginPaddingMinus30}.Layout(gtx, func(gtx C) D {
+														return layout.Flex{Alignment: layout.Start}.Layout(gtx,
+															layout.Flexed(1, func(gtx C) D {
+																gtx.Constraints.Max.X = 40
+																// TODO: Find a way to center this vertically.
+																return layout.Inset{
+																	Top:  values.MarginPadding30,
+																	Left: values.MarginPadding10,
+																}.Layout(gtx, func(gtx C) D {
+																	return pg.Theme.Icons.ActionInfo.Layout(gtx, pg.Theme.Color.Gray1)
+																})
+															}),
+															layout.Flexed(7, func(gtx C) D {
+																return layout.Inset{
+																	Top:  values.MarginPadding10,
+																	Left: values.MarginPadding10,
+																}.Layout(gtx, func(gtx C) D {
+																	return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+																		layout.Rigid(func(gtx C) D {
+																			label := pg.Theme.H6(values.String(values.StrSetUpStakeShuffleManualA))
+																			return label.Layout(gtx)
+																		}),
+																		layout.Rigid(func(gtx C) D {
+																			label := pg.Theme.Body1(values.String(values.StrSetUpStakeShuffleManualB))
+																			return label.Layout(gtx)
+																		}),
+																	)
+																})
+															}),
+														)
+													})
+												})
+											})
+										}),
+										layout.Rigid(func(gtx C) D {
+											return layout.Spacer{Height: 40}.Layout(gtx)
+										}),
+									)
 								}),
 								layout.Rigid(func(gtx C) D {
-									return layout.Inset{Top: values.MarginPadding10, Left: values.MarginPadding16, Right: values.MarginPadding16}.Layout(gtx, func(gtx C) D {
-										return layout.Flex{
-											Axis: layout.Horizontal,
-										}.Layout(gtx,
-											layout.Rigid(func(gtx C) D {
-												return pg.Theme.Icons.ActionInfo.Layout(gtx, pg.Theme.Color.Gray1)
-											}),
-											layout.Rigid(func(gtx C) D {
-												txt := `<span style="text-color: grayText2">
-											<b>Make sure to select the same accounts from the previous privacy setup. </b><br>Failing to do so could compromise wallet privacy.<br> You may not select the same account for mixed and unmixed.
```

Comment:
While this would allow to bold text in the middle of a sentence, the font ends up being different from the rest of the page so not using. Could also make everything html...